### PR TITLE
Improved edits to the `README.md` file

### PR DESCRIPTION
**Pull request by cr2007**
#### Reference Issue
None

#### What does this implement/fix? Explain your changes.
This implementation improves the overall layout of the README file
- Improves the Heading
- Adds a link to the logo
- Center-Aligns both the logo and heading
- Improves the Index with sub-bullets as well for the user to go directly to the point they need.
- Adds a **Back to Top ↥** button (aligned to the right) to go back to the top when the user is at the bottom of the file

#### Any other comments?
Feel free to add any more changes to these commits as you please.

Regards,
[**@cr2007**](https://github.com/cr2007)

### CODE REVIEW

**Comment by bmcfee** [comment]
Thanks for this!  Would you mind rolling back the alignment bits that use inlined HTML?  I'd prefer to keep things strict markdown here as noted in #1473.  The README pulls double-duty here as our splash page on the github repository, and the package description in [pypi](https://pypi.org/project/librosa/).

**Comment by cr2007** [comment]
Hey @bmcfee !

Sure, that can be possible
Although as you have written in #1473 , how does it render when it appears in the package description? in [pypi](https://pypi.org/project/librosa/)?

As center-aligning the logo, header make it easier for them to read
and putting the Back To Top button aligned to the right make it much easier to catch the user's attention amidst the long text.

If you could maybe share a screenshot within this thread of how it looks like, it'd help get a better idea perhaps.

If it doesn't go well, I can change those <html> elements to a (similar) Markdown state.

Regards,
@cr2007

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
-![librosa logo](docs/img/librosa_logo_text.svg)
+<div align="center">
+
+[![librosa logo](docs/img/librosa_logo_text.svg)](https://librosa.org/)
+
+# librosa
```
Comment (lostanlen) [inline README.md:5]: i'd be in favor of removing this mention of "librosa", which looks like a duplicate of the logo and takes up unnecessary space.

File: README.md [hunk 2]
```diff
@@ -120,7 +140,7 @@ conda install -c conda-forge ffmpeg
 
 If you are not using Anaconda, here are some common commands for different operating systems:
 
-- ###  Linux (apt-get): 
+- ####  Linux (`apt-get`):
```
Comment (lostanlen) [inline README.md:143]: nice

**Review by lostanlen — CHANGES_REQUESTED** [review]

**Comment by lostanlen** [comment]
Please disregard my review. I hadn't considered the importance of handling PyPI README

File: README.md (hunk 1 above)
Comment (cr2007) [reply README.md:5]: The first one is the logo (with a [hyperlink](https://librosa/org/) added if the user clicks on it)
And the second one is the Heading of the repository.

So I'm not really sure what the suggestion is here.

**Comment by bmcfee** [comment]
> Although as you have written in https://github.com/librosa/librosa/pull/1473 , how does it render when it appears in the package description? in [pypi](https://pypi.org/project/librosa/)?

The short answer is that I don't know, and I'd prefer to not deploy a test package just to find out.

According to the [official docs](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/), [github-flavored markdown](https://github.github.com/gfm/) is supported as the default markdown.  While that does techinically allow for inlined HTML, I'd still prefer to avoid it.

**Comment by cr2007** [comment]
Hi @bmcfee ,

I have done the necessary changes as you had asked in the next commit following your comment.

Let me know if there is anything else that is required.

Regards,
@cr2007

**Review by bmcfee — APPROVED** [review]

**Comment by bmcfee** [comment]
This looks fine to me.  Thanks again.

**Pull request merged**